source/docs/advanced/theme/sitemap.md

```diff
@@ -0,0 +1,172 @@
+---
+id: sitemap
+title: Configure the sitemap
```

Comment:
The page is more than just a configuration overview. It provides details about how sitemaps are generated too (configuring a CRON task…).

```suggestion
title: Sitemap generation
```